Classes/common/CDTIncrementalStore.m

```diff
@@ -16,6 +16,14 @@
 #import "CDTFieldIndexer.h"
 #import "CDTISGraphviz.h"
 
+@interface CDTISBatchUpdateResult : NSPersistentStoreResult
+@property (nonatomic, strong) id result;
+@property (nonatomic, assign) NSBatchUpdateRequestResultType resultType;
```

Comment:
Ok, adding this seems to solve that problem:
```objc
@implementation NSBatchUpdateResult
@synthesize result = _result;
@synthesize resultType = _resultType;

- (void)setResult:(id)result {
    _result = result;
}


- (id)result {
    return _result;
}

- (void)setResultType:(NSBatchUpdateRequestResultType)resultType {
    _resultType = resultType;
}

- (NSBatchUpdateRequestResultType)resultType {
    return _resultType;
}
@end
```

Now, I'm not sure this is all worth it but the cast we currently have makes me nervous.